.github/workflows/dotnet-format.yml

```diff
@@ -23,8 +23,12 @@ jobs:
     - name: Setup dotnet
       uses: actions/setup-dotnet@v3
 
-    - name: Install format tool
-      run: dotnet tool install -g dotnet-format
+    - name: Install dependencies
+      run: dotnet restore
+
+    - name: Protobuf compile
```

Comment:
Can we exclude generated files when running `dotnet format` instead?

Reply:
Good question. `dotnet format` actually excludes generated files by default. There is a switch `--include-generated` if you want to include them. Why I added this is the code consuming the generated protobuf classes blows up if they aren't there 😢 Here's an example of how it was failing before: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/actions/runs/6042774648/job/16398572729?pr=1339#step:4:989